ballerina/Module.md

```diff
@@ -93,7 +93,7 @@ compromising the correctness of the results and lays the foundation for parallel
 using multiple consumers within the same group each reading and processing data from a subset of topic partitions and 
 running in a single thread.
 
-Topic partitions are assigned to consumers automatically or you can manually assign topic partitions.
+Topic partitions are assigned to consumers automatically, or you can manually assign topic partitions.
```

Comment:
```suggestion
Topic partitions are assigned to consumers automatically or you can manually assign topic partitions.
```

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/377/commits/d8e897a4fcf01ef0602e9e87794a77441a554f65